SCSS/_media-queries.scss

```diff
@@ -0,0 +1,158 @@
+@media only screen and (max-width : 992px) {
+    .nav-link {
+        font-size: 16px!important;
+        line-height: 65px!important;
+        color: red;
+    }
+
+    .advantages-margin {
+        height: 160px;
+        width: 160px;
+        border: 7px solid $primary-color;
+        padding: 8px;
+        font-size: 13px;
+        color: #333d40;
+        cursor: all-scroll;
+    }
+
+    .advantages--box__inner {
+        border: 1px solid rgba(219, 114, 13, 0.8);
+    }
+
+    .advantages-margin .advantages--box__inner .advantages--icon {
```

Comment:
czy potrzebujemy aż tak dokładnego selektora? może samo .advantages--icon będzie wystarczające? druga sprawa że nazwa "advantages-margin" wydaje mi się mało przyjazna - jeżeli margines jest pewnego rodzaju modyfikatorem wtedy powinno być "advantages--margin", jeżeli natomiast "advantages-margin" to całościowa nazwa - w takim wypadku kolejna klasa wewnątrz powinna być "advantages-margin--box__inner"

Reply:
dokładność wynika z zagnieżdżenia w oryginalnym pliku _advantages.scss, inaczej nie da się tego nadpisać